appservice/src/createAppService/AppServicePlanCreateStep.ts

```diff
@@ -84,3 +85,13 @@ interface ExtendedAppServicePlan extends WebSiteManagementModels.AppServicePlan
         id: string;
     }
 }
+
+export function getPlanKind(wizardContext: IAppServiceWizardContext): string {
+    if (wizardContext.customLocation) {
+        return 'linux,kubernetes';
+    } else if (wizardContext.newSiteOS === WebsiteOS.linux) {
+        return WebsiteOS.linux;
+    } else {
+        return AppKind.app;
+    }
+}
```

Comment:
Can it be moved to AppKind.ts file? Is there a specific reason for this to be here?

Reply:
It's only used in this file, so I thought it made the most sense to be in this file (I can remove the "export"). IMO, it should only go in AppKind.ts if it's used by other files